hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient/src/test/java/org/apache/hadoop/fs/TestDFSIO.java

```diff
@@ -116,6 +122,8 @@ public class TestDFSIO implements Tool {
       "test.io.block.storage.policy";
   private static final String ERASURE_CODE_POLICY_NAME_KEY =
       "test.io.erasure.code.policy";
+  private ExecutorService excutorService = Executors.newFixedThreadPool(
```

Comment:
I suggest you use `CompletionService` here. Then we will not need to use a `CountDownLatch`.

Reply:
Thanks sir, very nice suggestion. Have updated it.